java/piranha/config/properties.json

```diff
@@ -41,5 +41,12 @@
   "linkURL": "<provide_your_url>",
   "annotations": [
     "ToggleTesting"
+  ],
+  "testMethodProperties" : [
```

Comment:
This will act as the configuration for the test methods that needs to be cleanup. Structure is the same as `methodProperties`

Reply:
Just change the flagType for the method. So, 
```
{
      "methodName": "when",
      "flagType": "mock",
      "argumentIndex": 0
 }
```